setup.py

```diff
@@ -16,8 +16,9 @@
     author='Ben Johnson',
     author_email='bkitej@gmail.com',
     packages=find_packages(),
+    python_requires=">=3.8",
     install_requires=[
-        'pickle5',
+        'jupyter',
```

Comment:
Removed. It runs without Jupyter though.

I've been getting frustrated with the interface when trying to clear the cache, might write some magics just to make it a bit easier to work with